.github/workflows/e2e.yml

```diff
@@ -0,0 +1,27 @@
+name: e2e tests
+
+on:
+  push:
+    branches:
+      - main
+      - release-*
+
+  pull_request:
+    branches:
+      - main
+      - release-*
+
+jobs:
+  e2e:
+    runs-on: [capmox,e2e,dcd-playground]
+    env:
+      PROXMOX_URL: ${{ secrets.PROXMOX_URL }}
+      PROXMOX_TOKEN: ${{ secrets.PROXMOX_TOKEN }}
+      PROXMOX_SECRET: ${{ secrets.PROXMOX_SECRET }}
+    steps:
+      - uses: actions/checkout@v4.1.1
```

Comment:
I think we'll need `pull_request_target` here. Otherwise this will always fail in PRs from forks due to the unavailable secrets.

Reply:
Good point.